<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgFilterBanner</class>
 <widget class="QWidget" name="WdgFilterBanner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>35</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QLabel" name="lText">
     <property name="text">
      <string>Filter activated - not all data is displayed</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="tbClearFilter">
     <property name="text">
      <string>Clear filter</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>169</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QToolButton" name="tbDontShowAgain">
     <property name="text">
      <string>Don't show again</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="tbClose">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
